src/main/java/org/opentripplanner/routing/api/request/via/VisitViaLocation.java

```diff
@@ -0,0 +1,102 @@
+package org.opentripplanner.routing.api.request.via;
+
+import java.time.Duration;
+import java.util.List;
+import java.util.Objects;
+import javax.annotation.Nullable;
+import org.opentripplanner.framework.geometry.WgsCoordinate;
+import org.opentripplanner.framework.time.DurationUtils;
+import org.opentripplanner.framework.tostring.ToStringBuilder;
+import org.opentripplanner.transit.model.framework.FeedScopedId;
+
+/**
+ * A visit-via-location is a physical visit to one of the stops or coordinates listed. An on-board
+ * visit does not count , the traveler must alight or board at the given stop for it to to be
+ * accepted. To visit a coordinate, the traveler must walk(bike or drive) to the closest point in
+ * the street network from a stop and back to another stop to join the transit network.
+ * <p>
+ * TODO: NOTE! Coordinates are NOT supported jet.
+ */
```

Comment:
```suggestion
 * A visit-via-location is a physical visit to one of the stops or coordinates listed. An on-board
 * visit does not count. The traveler must alight or board at the given stop for it to to be
 * accepted. To visit a coordinate, the traveler must walk(bike or drive) to the closest point in
 * the street network from a stop and back to another stop to join the transit network.
 * <p>
 * TODO: NOTE! Coordinates are NOT supported yet.
 */
```

Reply:
I remember we discussed this as well earlier, is it really that the coordinate can't be connected to the origin or destination without transit in-between? If so, this information should also be communicated in the API schema.